### DISCUSSION TITLE
How to see progress and which parameters to use.

### DISCUSSION THREAD

**Discussion by Ma1Dong** [projectdiscovery/nuclei#3095] [Q&A] [answered]
How to see progress and which parameters to use。For example, I want to know what percent of the scan is

Just like this，thanks very much
![image](https://user-images.githubusercontent.com/52943794/209774146-889e35c2-2a62-4750-b0f0-96eaab16a8db.png)

**Comment by ehsandeep** [comment, marked as answer]
@Ma1Dong you can use `-stats` option for this.

**Reply by Ma1Dong** [reply]
Thank you
